src/components/form/BetterDatePicker.tsx

```
@@ -0,0 +1,239 @@
+      return "success";
+    }
+    if (isSameMonth(day, displayedMonth)) {
+      return "outline-secondary";
+    }
+    return "link";
+  };
+
+  const updateDisplayedYear = (value: string) => {
+    if (value.length > 4) return;
+    const year = value ? parseInt(value) : 0;
+    if (isNaN(year)) return;
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)